packages/client/src/components/DateRangePicker.tsx

```diff
@@ -667,7 +667,10 @@ function DateRangePickerComponent(props: IDateRangePickerProps) {
       </PickerButton>
       {modalVisible && (
         <>
-          <ModalContainer id="picker-modal">
+          <ModalContainer
```

Comment:
I don't think we can make a change like this in here as it affects all components using this 🤔

Reply:
okay